include/z64actor.h

```diff
@@ -1183,4 +1183,38 @@ typedef enum {
     /* 0xC8 */ CLEAR_TAG_SMOKE = 200
 } ClearTagType;
 
+typedef enum {
+    /* 0x02 */ TATL_HINT_ID_MINI_BABA = 0x02,
+    /* 0x04 */ TATL_HINT_ID_MAD_JELLY,
+    /* 0x04 */ TATL_HINT_ID_SKULLTULA = 0x04,
+    /* 0x09 */ TATL_HINT_ID_WILTED_DEKU_BABA = 0x09,
+    /* 0x0B */ TATL_HINT_ID_NEJIRON = 0x0B,
+    /* 0x0E */ TATL_HINT_ID_DEEP_PYTHON = 0x0E,
+    /* 0x11 */ TATL_HINT_ID_FIRE_KEESE = 0x11,
+    /* 0x12 */ TATL_HINT_ID_KEESE,
+    /* 0x13 */ TATL_HINT_ID_ARMOS,
+    /* 0x16 */ TATL_HINT_ID_SHABOM = 0x16,
+    /* 0x19 */ TATL_HINT_ID_WART = 0x19,
+    /* 0x1F */ TATL_HINT_ID_SKULLWALLTULA = 0x1F,
+    /* 0x20 */ TATL_HINT_ID_GOLD_SKULLTULA,
+    /* 0x28 */ TATL_HINT_ID_GYORG = 0x28, // "If you go near it..."
+    /* 0x29 */ TATL_HINT_ID_GYORG_STUNNED, // "What are you doing?!""
```

Comment:
For these two, I think it's better to give a general summary rather than quote pieces of it. So the first one is something like "if you go near, you'll be eaten!" and the second one is something like "Jump in and attack it!"